### PR TITLE
Rename `isAction` to `isMarketplaceAction`

### LINKED ISSUE #41: Rename `isActions`
There are 2 kind of actions.

1. Marketplace actions
2. Repo Actions

`isMarketplaceActions`?

I think that `isActions` should be the repo one and have `isMarketplaceActions` as an addition

WDYT?

The current actions
https://github.com/fregante/github-url-detection/blob/0f8a4260f3f869c537b2e9f878ff2f712c4a363d/index.ts#L448-L453

### PR DESCRIPTION
**Pull request by yakov116**
Closes #41

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -445,13 +445,17 @@ export const hasCode = (url: URL | Location = location): boolean => // Static co
...
 	'https://github.com/marketplace/actions/urlchecker-action',
 	'https://github.com/marketplace/actions/github-action-for-assignee-to-reviewer',
 	'https://github.com/marketplace/actions/hugo-actions',
 ]);
 
+/** @deprecated use isMarketplaceAction instead */
+export const isActions = isMarketplaceAction;
+collect.set('isActions', combinedTestOnly);
+
 /** @deprecated use isActions instead */
```
Comment (yakov116) [inline index.ts:459]: This was supposed to be dropped in the last major

Comment (fregante) [reply index.ts:459]: D'oh!

Comment (fregante) [reply index.ts:459]: Ok let's clean this up and I'll release v4

Comment (yakov116) [reply index.ts]: Merge this then I will do the other 2

Comment (fregante) [reply index.ts]: You can create a single Actions-related PR, it's easier

**Review by fregante — APPROVED** [review]
Merge at will

**Pull request merged**